src/routes/admin/menu/customization/+page.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  import { customizationKeys, customizationsByKey } from "$stores/menuStore";
+</script>
+
+<h1 class="mb-8 text-2xl">Tilpasninger</h1>
```

Comment:
```suggestion
<a href="/admin/menu" class="btn w-full">Rediger meny</a>
<div class="divider"></div>
<h1 class="mb-8 text-2xl">Tilpasninger</h1>
```

Gå tilbake med samme knapp som man brukte for å komme hit.